Automattic/jetpack
# Update jetpack-connection-banner to Emerald style

This PR introduces revamped Jetpack Connection banner to match the recently introduced Emerald style. The new banner layout is strongly based on the current Jetpack Recommendations banner implementation.

Additionally, this PR cleans up banner styles and scripts that are not used anymore. It aims to declutter the banners' logic and facilitate the creation of any other banners in Emerald style in the future.

#### Changes proposed in this Pull Request:
- Update the Jetpack Connection banner to the Emerald style (same as the Recommendations banner)
- Clean up banners styles and scripts that are no longer used

#### Jetpack product discussion
p6TEKc-6dA-p2

#### Testing instructions:

##### Test a default banner
* Checkout this branch `git checkout update/jetpack-connection-banner-style`
* Run the WordPress local env with the jetpack plugin
* Run `jetpack watch` and select `plugins > jetpack` in order to build the new code introduced in this PR
* Ensure you're disconnected from WordPress in your current env
* Go to the dashboard `/wp-admin/index.php`
* You should see a new banner:
<img width="1490" alt="image" src="https://user-images.githubusercontent.com/8419292/170684705-fe0fb428-4a3c-4b2f-b779-917715095dfb.png">

* Go to plugins page `/wp-admin/plugins.php`
* You should see the same banner as well:
<img width="1714" alt="image" src="https://user-images.githubusercontent.com/8419292/170684805-59412d24-8fb2-4857-9835-87fc8c3d7016.png">

* Clicking "Set up Jetpack" should connect you to Jetpack; the banner should not display anymore after that point

##### Test a license aware banner
* Disconnect from Jetpack
* Replace the line 149 with `add_action( 'admin_notices', array( $this, 'render_license_aware_banner' ) );` to force license aware banner to display
* Go to the dashboard `/wp-admin/index.php`
* You should see the banner in Emerald style with a different copy attached:
<img width="1713" alt="image" src="https://user-images.githubusercontent.com/8419292/170686838-de84a242-beae-42a1-8ba1-1c001b5bdb38.png">

#### Does this pull request change what data or activity we track or use?

No.
